.travis.yml

```diff
@@ -6,7 +6,7 @@ cache:
 notifications:
   email: false
 node_js:
-  - '4'
+  - stable
```

Comment:
[Looks like there is nothing called `latest` the build had been failing for it](https://travis-ci.org/sarbbottam/eslint-rule-finder/jobs/121509817#L139)